temporal-sdk/src/main/java/io/temporal/internal/replay/ReplayWorkflowRunTaskHandler.java

```diff
@@ -264,13 +268,29 @@ private void applyServerHistory(WorkflowHistoryIterator historyIterator) {
           timerStopped = true;
         }
       }
+      if (!replayFailed) {
```

Comment:
How can `replayFailed` ever be `true` on this line?

Reply:
you're right that was an error